pavelib/paver_tests/test_paver_bok_choy_cmds.py

```diff
@@ -167,3 +169,72 @@ def test_invalid_verbosity_and_processes(self):
         suite = BokChoyTestSuite('', num_processes=2, verbosity=3)
         with self.assertRaises(BuildFailure):
             BokChoyTestSuite.verbosity_processes_string(suite)
+
+
+class TestPaverPa11ycrawlerCmd(unittest.TestCase):
+
+    """
+    Paver pa11ycrawler command test cases.  Most of the functionality is
+    inherited from BokChoyTestSuite, so those tests aren't duplicated.
+    """
+
+    def setUp(self):
+        super(TestPaverPa11ycrawlerCmd, self).setUp()
+
+        # Mock shell commands
+        mock_sh = patch('pavelib.utils.test.suites.bokchoy_suite.sh')
+        self._mock_sh = mock_sh.start()
+
+        # Cleanup mocks
+        self.addCleanup(mock_sh.stop)
+
+    def _expected_command(self, report_dir):
+        """
+        Returns the expected command to run pa11ycrawler.
+        """
+        cms_start_url = (
+            "http://localhost:8031/auto_auth?redirect=true&course_id=course-v1"
```

Comment:
nit: store the url-encoded coursename as a var. It'll probably be updated as the demo course grows. (maybe.)